nusacrowd/utils/constants.py

```diff
@@ -76,6 +76,7 @@ class Tasks(Enum):
 TASK_TO_SCHEMA = {
     Tasks.DEPENDENCY_PARSING: "KB",
     Tasks.WORD_SENSE_DISAMBIGUATION: "KB",
+    Tasks.WORD_SENSE_DISAMBIGUATION: "TEXT",
```

Comment:
Will the `TEXT`schema be generalized to all `Tasks.WORD_SENSE_DISAMBIGUATION`? If yes, could we remove the previous `kb` schema assigned to `Tasks.WORD_SENSE_DISAMBIGUATION`?

Reply:
Based on my searching for `Tasks.WORD_SENSE_DISAMBIGUATION` still just `Indonesian WSD` dataloader that implemented it. I guess for now can remove mapping between `Tasks.WORD_SENSE_DISAMBIGUATION` and `kb` schema. Let me fix it

<img width="275" alt="Screen Shot 2022-09-24 at 21 14 08" src="https://user-images.githubusercontent.com/11547086/192102865-9ced37b7-940c-425f-9cfe-adad9fa801d4.png">

Reply:
updated, kindly check again @bryanwilie, thank you!